src/Middleware/SpaServices/src/Prerendering/PrerenderTagHelper.cs

```diff
@@ -102,7 +102,8 @@ public override async Task ProcessAsync(TagHelperContext context, TagHelperOutpu
             if (!string.IsNullOrEmpty(result.RedirectUrl))
             {
                 // It's a redirection
-                ViewContext.HttpContext.Response.Redirect(result.RedirectUrl);
+                var permanentRedirect = renderResult.StatusCode.GetValueOrDefault() == 301;
```

Comment:
This doesn't look right. `renderResult` isn't defined anywhere in the file. Should it be `result`?

Reply:
Yes.